packages/webdriverio/src/utils.ts

```diff
@@ -0,0 +1,189 @@
+import type {
+  AxeResults,
+  ElementContext,
+  PartialResult,
+  ContextObject
+} from 'axe-core';
+import type {
+  BrowserObject,
+  AxeRunPartialParams,
+  AxeGetFrameContextParams,
+  AxeRunLegacyParams,
+  AxeFinishRunParams,
+  AxeSourceInjectResponse,
+  AxeSourceInjectParams
+} from './types';
+
+/**
+ * Validates that the client provided is WebdriverIO v5 or v6.
+ * @param {BrowserObject} client
+ * @returns {boolean}
+ */
+
+export const isWebdriverClient = (client: BrowserObject): boolean => {
+  if (!client || typeof client !== 'object') {
+    return false;
+  }
+
+  if (typeof client.execute !== 'function') {
+    return false;
+  }
+
+  if (typeof client.switchToFrame !== 'function') {
+    return false;
+  }
+
+  return true;
+};
+
+/**
+ * Get running context
+ * @param {Array} include
+ * @param {Array} exclude
+ * @returns {(ElementContext | null)}
```

Comment:
See this is why I don't think we should document our params / returns. If we don't pay close attention to them they go out of sync with TypeScript.

```suggestion
 * @returns {ElementContext}
```